opm/output/eclipse/Summary.cpp

```diff
@@ -77,13 +77,14 @@ struct quantity {
  * All functions must have the same parameters, so they're gathered in a struct
  * and functions use whatever information they care about.
  *
- * ecl_wells are wells from the deck, provided by opm-parser. wells is
- * simulation data
+ * ecl_wells are wells from the deck, provided by opm-parser. active_index is
+ * the index of the block in question. wells is simulation data.
  */
 struct fn_args {
     const std::vector< const Well* >& ecl_wells;
```

Comment:
As in [this well](https://github.com/OPM/opm-parser/blob/master/opm/parser/eclipse/EclipseState/Schedule/Well.hpp) - maybe `schedule_wells` would be a better name?